README.md

```diff
@@ -38,7 +38,6 @@ and API can change at any time.
 
 ### ⚙️ Discussions and explanations
 
-- [Project history][usr-expl-history]
```

Comment:
This rationale is not avoid duplication with the nope.website repo right ?

Reply:
Yeah it's annoying to keep both file in sync; I think I actually already adjusted a few things on one side and forgot to backport them already.

The website is public and on git as well so I don't see the benefit of the duplication.